src/server/imu-calibration.md

```diff
@@ -14,15 +14,15 @@ Other IMUs, such as the BNO085 or ICM20948 do not require any specific manual ca
 1. Upon flipping the IMU over, calibration should begin. To successfully calibrate your IMU you need to gently rotate the IMU in all 3 axes.
 1. After approximately 60 seconds has passed, the tracker should be successfully calibrated and will begin to show rotation in the SlimeVR server.
 
-## BMI160 with firmware v0.3.3 and above
+## BMI160 with firmware v0.3.3 and above, BMI270, LSM or other sfusion sensors with SlimeVR/main
 
-If you have a BMI160 and firmware v0.3.3 or higher, you will need to calibrate your IMU in a different way, unless specified otherwise in the firmware:
+If you have a BMI160 and firmware v0.3.3 or higher, or you are using BMI270, LSM6DSO, LSM6DSV or other sfusion imus on SlimeVR/Main or sfusion-tuned-mbe you will need to calibrate your IMU in a different way, unless specified otherwise in the firmware:
```

Comment:
```suggestion
If you have a BMI160 and firmware v0.3.3 or higher, or you are using BMI270, LSM6DSO, LSM6DSV or other sfusion IMUs on SlimeVR/Main or sfusion-tuned-mbe you will need to calibrate your IMU in a different way, unless specified otherwise in the firmware:
```